src/protocol/creation/routes.rs

```diff
@@ -89,6 +89,13 @@ pub async fn create_file(
         }
     }
 
+    if let Some(max_file_size) = state.config.max_file_size {
```

Comment:
Can you please move it below check that length is not None; Under the line 120.

Reply:
Ok. check pls